src/key_value_store/server/kvstore_server.h

```diff
@@ -0,0 +1,37 @@
+#include <grpcpp/grpcpp.h>
+
+#include "store.grpc.pb.h"
+
+#include "../core/KeyValueStore.h"
+
+using grpc::Server;
+using grpc::ServerBuilder;
+using grpc::ServerContext;
+using grpc::ServerReaderWriter;
+using grpc::Status;
+using grpc::StatusCode;
+
+using kvstore::PutRequest;
+using kvstore::PutReply;
+using kvstore::GetRequest;
+using kvstore::GetReply;
+using kvstore::RemoveRequest;
+using kvstore::RemoveReply;
+
+class KeyValueStoreImpl final :
```

Comment:
Put on one line.  (You may want to run clang-format.)

Reply:
done, ran this on all files